testsuite/tests/kuadrant/authorino/operator/http/test_raw_http.py

```diff
@@ -0,0 +1,18 @@
+"""
+Test raw http authorization interface.
+"""
+
+import pytest
+
+def test_authorized_via_http(authorization, client_http_auth, auth):
+    response = client_http_auth.request("GET", "/check", auth=auth, json='{}')
+    assert response.is_success
+    assert response.status_code == 200
+    assert response.text == ''
+    assert response.headers.get('x-ext-auth-other-json', '') == '{"propX":"valueX"}'
+
+def test_unauthorized_via_http(authorization, client_http_auth):
+    response = client_http_auth.request("GET", "/check", json='{}')
```

Comment:
`client_http_auth.get`

Reply:
`TypeError: Client.get() got an unexpected keyword argument 'json'`

Reply:
Why are you sending empty JSON in a get request?

Reply:
See this example:
```kubectl port-forward services/authorino-authorino-authorization 3000:3000 &;PID=$!

curl -H 'Content-type: application/json' -d '{}' http://wristband.127.0.0.1.nip.io:3000/check -i
# HTTP/1.1 200 OK

kill $PID``` from https://github.com/Kuadrant/authorino-operator/pull/61